config.go

```diff
@@ -40,16 +40,6 @@ func LoadConf(path string) (MqttConf, InfluxDBConf, error) {
 		return MqttConf{}, InfluxDBConf{}, err
 	}
 
-	em := MqttConf{}
```

Comment:
Yes, sure. I should have described the issue already before.

With introducing a slice in the struct, we cannot do a soft comparison via `==` anymore as it won't work.
It would need to use [reflect.DeepEqual](https://golang.org/pkg/reflect/#DeepEqual) .

Instead of pulling in reflection, I noticed that this comparison will probably only catch a very small subset of problems (and just logs it to console instead of aborting) as
- the default example is already pre-filled
- any config parse error is catched already before
- if it is really empty, the application should actually use some defaults instead.

However, I can re-add using reflect.DeepEqual. However, I think that the benefit is very marginal.

Reply:
Is it okay or should I change?

Reply:
Sorry for late response. I agree to you.